upi/vsphere/variables.tf

```diff
@@ -1,153 +1,167 @@
+// phpIPAM variables
+
+variable "ipam" {
+  type        = string
+  description = "The IPAM server to use for IP management."
+  default     = ""
+}
+
+variable "ipam_token" {
+  type        = string
+  description = "The IPAM token to use for requests."
+  default     = ""
+}
+
 //////
 // vSphere variables
 //////
 
 variable "vsphere_server" {
-  type        = "string"
+  type        = string
   description = "This is the vSphere server for the environment."
 }
 
 variable "vsphere_user" {
-  type        = "string"
+  type        = string
   description = "vSphere server user for the environment."
 }
 
 variable "vsphere_password" {
-  type        = "string"
+  type        = string
   description = "vSphere server password"
 }
 
 variable "vsphere_cluster" {
-  type        = "string"
+  type        = string
   description = "This is the name of the vSphere cluster."
 }
 
 variable "vsphere_datacenter" {
-  type        = "string"
+  type        = string
   description = "This is the name of the vSphere data center."
 }
 
 variable "vsphere_datastore" {
-  type        = "string"
+  type        = string
   description = "This is the name of the vSphere data store."
 }
 
 variable "vm_template" {
-  type        = "string"
+  type        = string
   description = "This is the name of the VM template to clone."
 }
 
 variable "vm_network" {
-  type        = "string"
+  type        = string
   description = "This is the name of the publicly accessible network for cluster ingress and access."
   default     = "VM Network"
```

Comment:
Why does this have a default but others don't?

Reply:
@patrickdillon I am not sure but my guess is CI.

Reply:
oh I see. Network probably isn't specified in the cluster_profile. hm...